src/collective/easyform/upgrades/configure.zcml

```diff
@@ -0,0 +1,8 @@
+<configure xmlns="http://namespaces.zope.org/zope">
+  <include file="1001.zcml" />
```

Comment:
Can we have something like https://github.com/plone/plone.app.upgrade/blob/f42a09f47173aff76b338ba18ae9fff0dda3b207/plone/app/upgrade/v52/configure.zcml#L7
Which keeps the upgrade steps all together but has a separate include for the upgrade profiles.
Just to be standardish :)

Reply:
As discussed in the chat, I'd prefer the style here - same as here:  https://github.com/plone/plone.staticresources/tree/master/src/plone/staticresources/upgrades
It's easier to keep track on what's going on. IMO plone.app.upgrades is harder to understand.
However, this is a minor detail ...